src/main/resources/messages.properties

```diff
@@ -1,6 +1,9 @@
 # Provides a 'catch all' for errors when casting text inputs from strings to integers
 typeMismatch.java.lang.Integer=Enter a value without commas, brackets or other symbols
 
+# Accounts criteria
+isCriteriaMet=
+
```

Comment:
Is this message used anywhere?

Reply:
Yes, it's used when a binding error is triggered on `POST` calls in the global errors section. We don't display the name of the field, hence it's left as an empty string. Without this, Thymeleaf attempts to resolve the field name without success and the display isn't correct.

Reply:
Can this be handled in the template instead of adding empty properties (making the assumption there could be more occurrences of this)? If not, can you add a comment to explain why it's empty please.

Reply:
Using a ternary operator to check if the property is null, Thymeleaf still falters for some reason and the global error is formatted as:
```
??<message>_en_US??
```
I'll add a comment for now for clarity